lib/booking_manager.rb

```diff
@@ -0,0 +1,184 @@
+require_relative 'room'
+require_relative 'reservation'
+# require 'pry'
+
+module Hotel
+  class BookingManager
+    attr_accessor :rooms, :reservations, :room_calendar
+
+    def initialize(number_rooms)
+      @rooms = populate_room_list(number_rooms) #(20)
+      @reservations = make_reservation_list
+      @room_calendar = make_room_calendar(number_rooms)
```

Comment:
I like that you've broken each of these initialization helpers out as separate methods - the pattern is very clear.